File: Models/HwControllerMetadata.cs
using System;
using System.Text;
using System.Collections.Generic;


namespace OccupOSMonitor.Models {

    public class HwControllerMetadata : Entity {
        public HwControllerMetadata() {
            SensorDatas = new List<SensorData>();
            SensorMetadatas = new List<SensorMetadata>();
        }
        public virtual string ExternalId { get; set; }
        public virtual string DepartmentName { get; set; }
        public virtual string BuildingName { get; set; }
        public virtual System.DateTime UpdatedAt { get; set; }
        public virtual System.DateTime CreatedAt { get; set; }
        public virtual System.Nullable<int> UpdaterId { get; set; }
        public virtual System.Nullable<int> CreatorId { get; set; }
        public virtual System.Nullable<int> FloorNr { get; set; }
        public virtual string RoomId { get; set; }
        public virtual IList<SensorData> SensorDatas { get; set; }
        public virtual IList<SensorMetadata> SensorMetadatas { get; set; }
    }
}

File: app/scripts/models/HwControllerMetadata.js
OccupOS.HwControllerMedadata = DS.Model.extend({
    externalId:     DS.attr('string'),
    departmentName: DS.attr('string'),
    buildingName:   DS.attr('string'),
    floorNr:        DS.attr('number'),
    creatorId:      DS.attr('number'),
    updaterId:      DS.attr('number'),
    createdAt:      DS.attr('date'),
    updatedAt:      DS.attr('date'),
    sensorMetadata: DS.hasMany('OccupOS.SensorMetadata'),
    sensorData:     DS.hasMany('OccupOs.SensorData')
});